product.go

```diff
@@ -27,6 +27,8 @@ type ProductParams struct {
 	Name              string             `form:"name"`
 	PackageDimensions *PackageDimensions `form:"package_dimensions"`
 	Shippable         *bool              `form:"shippable"`
+	Statement         string             `form:"statement_descriptor"`
```

Comment:
can you name it `StatementDescriptor`? I'm working on renaming all properties in https://github.com/stripe/stripe-go/pull/459 to ensure they match, as best as possible, the API name